runtime/src/serde_snapshot/legacy.rs

```diff
@@ -8,6 +10,9 @@ pub(super) struct SerializableAccountStorageEntry {
     count_and_status: (usize, AccountStorageStatus),
 }
 
+#[cfg(all(test, RUSTC_WITH_SPECIALIZATION))]
+impl IgnoreAsHelper for SerializableAccountStorageEntry {}
```

Comment:
It would be nice to have some brief documentation on when `IgnoreAsHelper` should be used, perhaps in 
`docs/src/implemented-proposals/abi-management.md`

Reply:
I see! I'll do this as a follow-up pr.